templates/zerver/help/keyboard-shortcuts.md

```diff
@@ -25,7 +25,7 @@ below, and add more to your repertoire as needed.
 
 * **New stream message**: `c` — Start a new topic in the current stream.
 
-* **New private message**: `C`
+* **New private message**: `x` — Start a new private message.
```

Comment:
You'll need to reflect the C -> x migration in the Keyboard shortcuts help modal as well `templates/zerver/keyboard_shortcuts.html`